games-emulation/pcsxr/files/pcsxr-1.9.95-harfbuzz-dep-fix.patch

```diff
@@ -0,0 +1,79 @@
+new file mode 100644
+--- /dev/null
```

Comment:
Can we get something shorter for Gentoo? It's fine if you literally just had to pass paths via CMake.